packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -3152,16 +3137,9 @@ describe('KeyringController', () => {
       await withController(
         {
           cacheEncryptionKey: true,
-          skipVaultCreation: true,
-          state: {
-            vault: JSON.stringify({ data: '0x123', salt: 'my salt' }),
-            // @ts-expect-error we want to force the controller to have an
-            // encryption salt equal to the one in the vault
-            encryptionSalt: 'my salt',
-          },
```

Comment:
The concept here was similar to what we have for `submitPassword`: without these instructions (`skipVaultCreation` in particular) the controller will already be unlocked, so these assertions don't make particular sense now:
`expect(controller.state.isUnlocked).toBe(true);`

Reply:
reverted back